openms-wiki/building-openms.md

```diff
@@ -0,0 +1,11 @@
+# Build OpenMS
+
+To build OpenMS, please follow the build instructions:
+
+* [Instructions for Linux](instructions-for-linux.md)
+* [Instructions for OSX](instructions-for-osx.md)
+* [Instructions for Windows](instructions-for-windows.md)
```

Comment:
let's link these to the ones listed here: https://github.com/OpenMS/OpenMS/wiki/Building-OpenMS.